pkg/utils/utilconsts.go

```diff
@@ -23,8 +23,10 @@ const (
 	debugLevel = klog.Level(5)
 	// HelmCRKind is kind of the Helm CR
 	HelmCRKind = "HelmRelease"
+	// SubscriptionCRKind is kind of the Subscription CR
+	SubscriptionCRKind = "Subscription"
 	// HelmCRAPIVersion is APIVersion of the Helm CR
-	HelmCRAPIVersion = "app.ibm.com/v1alpha1"
+	HelmCRAPIVersion = "apps.open-cluster-management.io/v1"
```

Comment:
```go
import (
  chnv1 ""github.com/open-cluster-management/multicloud-operators-channel/pkg/apis/apps/v1"
)
HelmCRAPIVersioon = chnv1.SchemeGroupVersion.String()
```

Reply:
I cannot use chnv1.SchemeGroupVersion variable as a constant value.

Reply:
Then make it a var rather than a "const"? :)